### PR TITLE
Added documentation for new flag  in arangoexport

### PR DESCRIPTION
**Pull request by cpjulia**
Documentation for https://github.com/arangodb/arangodb/pull/15324

### CODE REVIEW

File: 3.10/release-notes-upgrading-changes310.md [hunk 1]
```diff
@@ -42,3 +42,9 @@ These test cases had been deprecated since ArangoDB 3.9.
 
 The testcase `hash` was renamed to `persistent-index` to better reflect its
 scope.
+
+### arangobench
+
+
+Added new option `--custom-query-bindvars` to arangoexport, so queries given via 
+option `--custom-query` can have bind variables in them.
```
Comment (jsteemann) [inline 3.10/release-notes-upgrading-changes310.md:50]:
```suggestion

### arangoexport

Re existing arangoexport option `--query` was renamed to `--custom-query`, for
improved naming consistency across different client tools.
Using the old option name `--query` is still supported and will implicitly use the option
`--custom-query` under the hood. Client scripts should eventually be updated to use
the new option name though.
```

File: 3.10/programs-arangoexport-examples.md [hunk 2]
```diff
@@ -134,19 +134,24 @@ This exports the named graph mygraph into the xgmml file *mygraph.xgmml* with a
...
+the arangoexport option `--custom-query-max-runtime`. This specifies the maximum query
 runtime in seconds. Set it to `0` for no limit.
 
-    arangoexport --type jsonl --query-max-runtime 10 --query "FOR book IN books FILTER book.sells > 100 RETURN book"
+    arangoexport --type jsonl --custom-query-max-runtime 10 --custom-query "FOR book IN books FILTER book.sells > 100 RETURN book"
+
+With the flag `--custom-query-bindvars`, the query provided via `--custom-query` can have bind variables.
+
+
+    arangoexport --type jsonl --custom-query 'FOR book IN @@@@collectionName FILTER book.sells == @@sells RETURN book' --custom-query-bindvars '{"@@collectionName": "books", "sells": 100}'
```
Comment (Simran-B) [inline 3.10/programs-arangoexport-examples.md:157]: Should mention that @ needs to be escaped with another @ somewhere above

Comment (jsteemann) [reply 3.10/programs-arangoexport-examples.md:157]: Oh, yes, definitely!

Comment (cpjulia) [reply 3.10/programs-arangoexport-examples.md:157]: Addressed

File: 3.10/release-notes-upgrading-changes310.md [hunk 3]
```diff
@@ -42,3 +42,14 @@ These test cases had been deprecated since ArangoDB 3.9.
 
 The testcase `hash` was renamed to `persistent-index` to better reflect its
 scope.
+
+### arangoexport
+
+
+Added new option `--custom-query-bindvars` to arangoexport, so queries given via 
+option `--custom-query` can have bind variables in them.
+Re existing arangoexport option `--query` was renamed to `--custom-query`, for
```
Comment (jsteemann) [inline 3.10/release-notes-upgrading-changes310.md:51]:
```suggestion
The existing arangoexport option `--query` was renamed to `--custom-query`, for
```

**Review by ansoboleva — APPROVED** [review]
reviewed

**Pull request merged**